src/Analyzer.Cli.UnitTests/AppServicesLogs-Failures.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
I was not sure if I should create the file in TestInit() method, then clean them up at the end. I think it is cleaner to have them be created there, but then that means for each test, it would re-create them (sometimes unnecessarily). I decided to then create them and have them be copied to output if newer.

Reply:
They can be created in ClassInit, then they should only be created once per test class instance.

Reply:
I meant instead of declaring these as files, I create a file, add the text to it, then delete is and the end of the test

Reply:
Deciding I will leave the test as it is.